xpdacq/xpdacq.py

```diff
@@ -386,9 +386,15 @@ def __call__(self, sample, plan, subs=None, *,
             raise ValueError("These keys in metadata_kw are illegal "
                              "because they are always in sample: "
                              "{}".format(set(sample) & set(metadata_kw)))
-        if self._beamtime.get('bt_wavelength') is None:
-            print("WARNING: there is no wavelength information in current"
-                  "beamtime object, scan will keep going....")
+        if self._beamtime.wavelength is None:
+            warnings.warn("no wavelength is set to current beamtime "
```

Comment:
No wavelength has been recorded for the current beamtime.\\
Obtain the correct wavelength from the instrument scientist and "